frontend/src/components/home/widgets/LightningMessageboardWidget.tsx

```diff
@@ -238,7 +238,7 @@ export function LightningMessageboardWidget() {
                     </CardTitle>
                     <div>
                       <Badge className="py-1">
-                        <ZapIcon className="w-4 h-4 mr-1" />{" "}
+                        <ZapIcon className="size-4 mr-1" />{" "}
```

Comment:
bottom spacing is missing now

<img width="1141" height="646" alt="image" src="https://github.com/user-attachments/assets/a62d3e73-65f6-4382-918e-40ca57710b49" />